### ISSUE TITLE
update to 1.9

### ISSUE THREAD

**Issue by hhucqian** [GuardRex/vscode-status-bar-tasks#17]
https://code.visualstudio.com/updates/v1_9#_task-support

now I can define different commands per task. and the extension can't work correctly

**Comment by guardrex** [comment]
Correct. I have to find some time this weekend to do the upgrade for the extension. I'll see if I can get it done in the next two days and ping you back here.

**Comment by guardrex** [comment]
I did the work on this, and I'm almost ready to update the extension. I'm just waiting on a answer from them on another issue: https://github.com/Microsoft/vscode/issues/17566

If I don't hear back from them within 24 hours, I'll go ahead and get the extension up there and deal with that other issue in a future update.

**Comment by guardrex** [comment]
I was just informed that the underlying problem with the `tasks.json` schema not picking up my new property is a rather difficult nut for them to crack. They opened a new issue to request the feature, but new features like this may take quite a long time or never become implemented. The new issue is at: https://github.com/Microsoft/vscode/issues/20193

I'm going to try to get this extension updated today (2/8), which will fix the bug on this issue.

**Comment by guardrex** [comment]
Just released v0.2.0 that fixes this issue.

You can add and use a `command` property on individual tasks. If you do so, the global `command` and `args` are not applied.

If you leave your command and arguments in the `args` property and don't specify a task `command`, then the global `command` and `args` will be used.

```json
{
    "version": "0.1.0",
    "command": "cmd",
    "isShellCommand": true,
    "showOutput": "silent",
    "args": [
        "/c"
    ],
    "test": "blah blah blah",
    "tasks": [
        {
            "taskName": "restore that draws in the global command and args",
            "suppressTaskName": true,
            "args" : ["dotnet", "restore"],
            "showOutput": "always",
            "problemMatcher": "$msCompile"
        },
        {
            "taskName": "restore that doesn't draw in the global command and args",
            "suppressTaskName": true,
            "command": "dotnet",
            "args" : ["restore"],
            "showOutput": "always",
            "problemMatcher": "$msCompile"
        },
```
The first restore command will run ...
```
cmd /c dotnet restore
```
The second restore command will run ...
```
dotnet restore
```
